arm/cdn/customdomains.go

```diff
@@ -118,14 +122,18 @@ func (client CustomDomainsClient) CreateResponder(resp *http.Response) (result a
 	return
 }
 
-// DeleteIfExists sends the delete if exists request.
+// DeleteIfExists sends the delete if exists request.This method handles
```

Comment:
Correction: User can cancel the polling, but LRO functionality will remain the same. If I call CreateOrUpdate with cancel channel, it will still be the same call. Only difference will be that sdk won't do polling this time.

I will change the comments.